### PR TITLE
Typo corrections

### PR DESCRIPTION
**Pull request by akshakp**
Updated typos for Openshift spelling under Openshift support section

### CODE REVIEW

File: docs/src/main/asciidoc/kubernetes-resources.adoc [hunk 1]
```diff
@@ -404,14 +404,14 @@ To enable the generation of Openshift resources, you need to include Openshift i
 kubernetes.deployment.target=openshift
 ----
 
-If you need to generate resources for both platforms (vanilla kubernetes and opesnhift), then you need to include both (coma separated).
+If you need to generate resources for both platforms (vanilla kubernetes and openshift), then you need to include both (coma separated).
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes-resources.adoc]:
```suggestion
If you need to generate resources for both platforms (vanilla Kubernetes and OpenShift), then you need to include both (coma separated).
```

File: docs/src/main/asciidoc/kubernetes-resources.adoc [hunk 2]
```diff
@@ -404,14 +404,14 @@ To enable the generation of Openshift resources, you need to include Openshift i
...
 
 [source]
 ----
 kubernetes.deployment.target=kubernetes, openshift
 ----
 
-The Openshfit resources can be customized in a similar approach with kubernetes.
+The Openshift resources can be customized in a similar approach with kubernetes.
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes-resources.adoc]:
```suggestion
The OpenShift resources can be customized in a similar approach with kubernetes.
```

**Review by gsmet — COMMENTED** [review]
Thanks! Let's get the case right while we're at it :).

Can you apply my suggestions and squash everything?

File: docs/src/main/asciidoc/kubernetes-resources.adoc (hunk 2 above)
Comment (akshakp) [reply docs/src/main/asciidoc/kubernetes-resources.adoc]: Thank you @gsmet . Added the suggestions :)

Comment (gsmet) [reply docs/src/main/asciidoc/kubernetes-resources.adoc]: Looks like you forgot this one? And can you squash everything in a single commit? Thanks!

Comment (akshakp) [reply docs/src/main/asciidoc/kubernetes-resources.adoc]: @gsmet updated most of the case references. Please confirm :)

**Review by gsmet — APPROVED** [review]
Looks good. I squashed your commits into one and will merge.

Thanks!

**Pull request merged**